dap_prinz_green_jobs/pipeline/ojo_application/extract_green_skills.py

```diff
@@ -0,0 +1,137 @@
+"""
+Extract green skills pipeline
+---------
+
+A pipeline that takes a sample of job adverts and extacts and maps
+skills onto a custom green skills taxonomy.
+
+(with default parameters):
+
+python dap_prinz_green_jobs/pipeline/ojo_application/extract_green_skills.py
+"""
+from dap_prinz_green_jobs.getters.data_getters import (
+    get_s3_resource,
+    save_to_s3,
+)
+from dap_prinz_green_jobs import BUCKET_NAME, PROJECT_DIR, logger
+
+from typing import List, Dict
+from tqdm import tqdm
+from itertools import islice
+from ojd_daps_skills.pipeline.extract_skills.extract_skills import (
+    ExtractSkills,
+)  # import the module
+from dap_prinz_green_jobs.getters.ojo import get_ojo_sample
+from datetime import datetime as date
+import argparse
+
+s3 = get_s3_resource()
+batch_size = 5000
+
+
+def format_extracted_green_skills(
+    job_ids: List[str],
+    skills: List[Dict[str, list]],
+    matched_skills: List[Dict[str, list]],
+) -> Dict[int, dict]:
+    """Format matched skills to flag extracted skills that do not
```

Comment:
i'm looking at the data and have just realised that i think something buggy is happening as the first half of the output is not quite right i.e. the wrong skill span corresponding to the mapped skill

Reply:
looking into it now + will modify